src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -38,52 +44,50 @@ let nextId: number = 0;
   animations: [skyAnimationSlide]
 })
 export class SkyRepeaterItemComponent implements OnDestroy {
-  public contentId: string = `sky-radio-content-${++nextId}`;
 
-  public get isExpanded(): boolean {
-    return this._isExpanded;
-  }
+  @Input()
+  public inlineFormConfig: SkyInlineFormConfig;
+
+  @Input()
+  public inlineFormTemplate: TemplateRef<any>;
 
   @Input()
   public set isExpanded(value: boolean) {
     this.updateForExpanded(value, true);
   }
 
-  public get isSelected(): boolean {
-    return this._isSelected;
+  public get isExpanded(): boolean {
+    return this._isExpanded;
   }
 
   @Input()
   public set isSelected(value: boolean) {
     this._isSelected = value;
   }
 
-  @Input()
-  public showInlineForm: boolean = false;
-
-  @Input()
-  public inlineFormConfig: SkyInlineFormConfig;
+  public get isSelected(): boolean {
+    return this._isSelected;
+  }
 
   @Input()
-  public inlineFormTemplate: TemplateRef<any>;
-
-  @Output()
-  public inlineFormClose = new EventEmitter<SkyInlineFormCloseArgs>();
+  public selectable: boolean = false;
 
   @Input()
-  public selectable: boolean = false;
+  public showInlineForm: boolean = false;
 
   @Output()
   public collapse = new EventEmitter<void>();
 
   @Output()
   public expand = new EventEmitter<void>();
 
-  public slideDirection: string;
+  @Output()
+  public inlineFormClose = new EventEmitter<SkyInlineFormCloseArgs>();
+
+  public active: boolean = false;
```

Comment:
Should we follow the pattern of the other public booleans and prefix with `is` (e.g., `isActive`)?

Reply:
I originally had `isActive`, but then switched it to be more like our newer components. I don't have strong feelings here, so I'll swap it back for consistency's sake.